# Removes march=native and replaces with EasyBuild-specific optimization flags
# adds -fpermissive because recent compilers don't like SpectrA (dependency) header code
diff -ru flashpca-2.0.orig/Makefile flashpca-2.0/Makefile
--- flashpca-2.0.orig/Makefile	2016-12-16 04:33:56.000000000 +0000
+++ flashpca-2.0/Makefile	2022-04-21 14:51:53.861068749 +0000
@@ -26,7 +26,7 @@
    CXXFLAGS += -msse2 -DEIGEN_DONT_PARALLELIZE -std=c++11
    BOOST = ${BOOST_LIB}/libboost_program_options.a
 else
-   CXXFLAGS += -march=native -fopenmp -std=c++0x
+   CXXFLAGS += ${OPTFLAGS} -fopenmp -std=c++0x -fpermissive
    BOOST = -L${BOOST_LIB} -lboost_program_options
 endif
 
